src/tools/pub/glossary.md

```diff
@@ -12,11 +12,10 @@ the [pub tool][].
 
 ## Application package
 
-A package that is not intended to be used as a library. Application packages may
-have [dependencies][] on other packages, but are never depended on
-themselves. They are usually meant to be run directly, either on the command
-line or in a browser. The opposite of an application package is a
-[library package][].
+A package that is not intended to be used as a library, unlike regular [packages][].
```

Comment:
```suggestion
A package that is not intended to be shared, unlike regular [packages][].
```

Individual libraries within an app package _are_ used like libraries - they just aren't intended for use outside the package, so I think just using the term "not intended to be shared" is better here?

Reply:
Thanks for explaining! I thought "library" and "application" were mutually exclusive for some reason (probably because I've been reading the same docs that confused our users 😅)